### DISCUSSION TITLE
Does go-zero converting decimals in MySQL to float64 cause issues? Can you help answer this question?

### DISCUSSION THREAD

**Discussion by x1rh** [zeromicro/go-zero#4338] [Q&A] [answered]
Does go-zero converting decimals in MySQL to float64 cause issues? Can you help answer this question?
（go-zero将MySQL的decimal转换为了float64是否会有精度问题，想问一下你们是怎么处理这个问题的？）

**Comment by kevwan** [comment, marked as answer]
You can use decimal. And config the conversion in goctl config file.
